src/sentry/audit_log/events.py

```diff
@@ -12,7 +12,9 @@ def __init__(self):
     def render(self, audit_log_entry: AuditLogEntry):
         if audit_log_entry.target_user == audit_log_entry.actor:
             return "joined the organization"
-        return f"add member {audit_log_entry.target_user.get_display_name()}"
+
+        member = audit_log_entry.data.get("email") or audit_log_entry.target_user.get_display_name()
```

Comment:
should we prioritize rendering with `get_display_name()` first if we have it and then fall back to email?

Reply:
I initially considered this approach, but the rest of the events favored email first. I don't totally understand the decision and am can switch it.

Reply:
let's keep it consistent then! email should also be easier to identify than display name anyway 🙌🏽